src/fights.ts

```diff
@@ -107,6 +109,7 @@ export function dailyFights() {
         }
 
         if (
+          SourceTerminal.have() &&
```

Comment:
unnecessary - this is just making sure there is no digitize monster up

Reply:
Addressed as of 15879b0